packages/image/src/main.vue

```diff
@@ -11,6 +11,7 @@
       class="el-image__inner"
       :src="src"
       :alt="alt"
+      :referrerPolicy="referrerPolicy"
```

Comment:
`:referrerPolicy=` -> `:referrerpolicy=` is better according to the html standard